src/mixins/common-mixin.ts

```diff
@@ -77,13 +76,13 @@ export class CommonMixin extends Vue {
       case EntityTypes.SO: return CorpTypeCd.SOCIETY
       case EntityTypes.UL: return CorpTypeCd.BC_ULC_COMPANY
       case EntityTypes.XCP: return CorpTypeCd.XPRO_COOP
-      case EntityTypes.XCR: return CorpTypeCd.XPRO_CORPORATION
+      case EntityTypes.XCR: return CorpTypeCd.EXTRA_PRO_A
```

Comment:
XCR === A

Reply:
Ah man. Again, these can be soo confusing. On paper it sounds like EXTRA_PRO_A is EntityTypes.A but apparently that's not the case. Just like how BC is BEN but CR is BC.

Reply:
😠